app/services/notification_service.py

```
@@ -124,18 +307,32 @@ async def create_and_dispatch_notification(
-        notifiers = db.query(Notifier).filter(Notifier.enabled).all()
+        Args:
+            db: Database session
+            event_type: Type of notification event
 
-        return [n for n in notifiers if level_value >= level_hierarchy[n.filter_level]]
+        Returns:
+            List of enabled notifiers subscribed to this event
+        """
+        # For SQLite, we need to do JSON parsing manually
+        # Get all enabled notifiers and filter in Python
+        all_notifiers = db.query(Notifier).filter(Notifier.enabled == True).all()
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `Notifier.enabled:` for truth checks

Replace with `Notifier.enabled`

(E712)